jmxtrans-cluster/src/test/java/com/googlecode/jmxtrans/cluster/TestUtils.java

```diff
@@ -0,0 +1,27 @@
+package com.googlecode.jmxtrans.cluster;
+
+import com.googlecode.jmxtrans.cluster.zookeeper.ZookeeperClusterService;
+import org.apache.commons.configuration.Configuration;
+import org.apache.commons.configuration.HierarchicalConfiguration;
+
+/**
+ * TestUtils. The place of common routines for cluster testing.
+ *
+ * @author Tibor Kulcsar
+ * @since <pre>May 25, 2016</pre>
+ */
+public class TestUtils {
```

Comment:
I would rename this in `ConfigurationFixtures`. TestUtils is very non descriptive as a name.